token-auth.yml

```
@@ -0,0 +1,174 @@
+openapi: '3.0.2'
+info:
+  title: Token Auth API
+  description: API reference for token authentication. 
+  contact:
+    email: support@getmentum.com
+  version: '0.1'
+
+servers:
+  - url: https://api.getmentum.com/v0
+    description: Production endpoint.
+  - url: https://sandbox.getmentum.com/v0
+    description: Sandbox environment.
+
+tags:
+  - name: Token Authentication
+    description: Use the endpoints below to create and manage your access tokens.
+
+components:
+  securitySchemes:
+    Basic_Authentication:
+      description: |
+        Mentum currently provides two authentication methods: **Basic Auth (via API Key Pairs)** and **JWT tokens**.
+        
+        Whenever using Basic Auth, we recommend using a secure communication protocol like **HTTPS** or **SSL**.
+
+        From the developer dashboard, generate an API Key Pair (**Key Id** and **Secret**) 
+        and use them as username and password respectively via Basic Authentication.
+      type: http
+      scheme: basic
+    JWT_Authentication:
+      description: |
+        JWT Tokens are generally used by front-end components such as Dynamic Widgets or Single Page Applications (SPA).
+
+        JWT tokens are generated by your API Key Pair (**Key Id** and **Secret**). You can generate a new API Key Pair via the developer dashboard.
+        
+        Use `/auth/accessToken/create` to exchange your API Key Pair for a JWT token.
+        
+        The JWT Access Token has an expiration of thirty minutes. You can refresh the expired JWT token via `/auth/accessToken/refresh` endpoint.
+      type: http
+      scheme: bearer
+      bearerFormat: JWT
+
+paths:
+  /api/accessToken/create:
```

```suggestion
  /auth/accessToken/create:
```

I believe we updated these paths right?